### PR TITLE
Haksoat model rebuild

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: model_info/arwiki.articletopic.md [hunk 1]
```diff
@@ -663,299 +663,299 @@ Model Information:
...
+		STEM.Libraries & Information                 0.97
+		STEM.Mathematics                             0.975
+		STEM.Medicine & Health                       0.972
+		STEM.Physics                                 0.98
+		STEM.STEM*                                   0.974
+		STEM.Space                                   0.989
+		STEM.Technology                              0.972
 		-------------------------------------------  -----
-	pr_auc (micro=0.772, macro=0.627):
+	pr_auc (micro=0.756, macro=0.591):
```
Comment (halfak) [inline model_info/arwiki.articletopic.md:892]: Minor loss in fitness.

File: model_info/cswiki.articletopic.md [hunk 2]
```diff
@@ -778,184 +778,184 @@ Model Information:
...
+		STEM.Libraries & Information                 0.968
+		STEM.Mathematics                             0.983
+		STEM.Medicine & Health                       0.974
+		STEM.Physics                                 0.981
+		STEM.STEM*                                   0.975
+		STEM.Space                                   0.993
+		STEM.Technology                              0.974
 		-------------------------------------------  -----
-	pr_auc (micro=0.798, macro=0.663):
+	pr_auc (micro=0.766, macro=0.599):
```
Comment (halfak) [inline model_info/cswiki.articletopic.md:892]: Minor loss in fitness

File: model_info/enwiki.articletopic.md [hunk 3]
```diff
@@ -554,408 +554,408 @@ Model Information:
...
+		STEM.Medicine & Health                       0.978
+		STEM.Physics                                 0.981
 		STEM.STEM*                                   0.977
-		STEM.Space                                   0.991
-		STEM.Technology                              0.972
+		STEM.Space                                   0.987
+		STEM.Technology                              0.969
 		-------------------------------------------  -----
-	pr_auc (micro=0.787, macro=0.649):
+	pr_auc (micro=0.788, macro=0.656):
```
Comment (halfak) [inline model_info/enwiki.articletopic.md:892]: Minor increase in fitness

File: model_info/enwiki.drafttopic.md [hunk 4]
```diff
@@ -684,278 +684,278 @@ Model Information:
...
+		STEM.Libraries & Information                 0.959
+		STEM.Mathematics                             0.968
+		STEM.Medicine & Health                       0.967
+		STEM.Physics                                 0.975
+		STEM.STEM*                                   0.968
+		STEM.Space                                   0.983
+		STEM.Technology                              0.958
 		-------------------------------------------  -----
-	pr_auc (micro=0.738, macro=0.584):
+	pr_auc (micro=0.729, macro=0.566):
```
Comment (halfak) [inline model_info/enwiki.drafttopic.md:892]: Minor loss in fitness

File: model_info/kowiki.articletopic.md [hunk 5]
```diff
@@ -519,443 +519,443 @@ Model Information:
...
+		STEM.Libraries & Information                 0.972
+		STEM.Mathematics                             0.981
+		STEM.Medicine & Health                       0.971
+		STEM.Physics                                 0.98
+		STEM.STEM*                                   0.974
+		STEM.Space                                   0.988
+		STEM.Technology                              0.971
 		-------------------------------------------  -----
-	pr_auc (micro=0.797, macro=0.66):
+	pr_auc (micro=0.775, macro=0.614):
```
Comment (halfak) [inline model_info/kowiki.articletopic.md:892]: Minor loss in fitness

File: model_info/viwiki.articletopic.md [hunk 6]
```diff
@@ -551,411 +551,411 @@ Model Information:
...
+		STEM.Libraries & Information                 0.961
+		STEM.Mathematics                             0.985
+		STEM.Medicine & Health                       0.974
+		STEM.Physics                                 0.986
+		STEM.STEM*                                   0.981
+		STEM.Space                                   0.989
+		STEM.Technology                              0.975
 		-------------------------------------------  -----
-	pr_auc (micro=0.826, macro=0.684):
+	pr_auc (micro=0.711, macro=0.533):
```
Comment (halfak) [inline model_info/viwiki.articletopic.md:892]: Non-minor loss in fitness.  Might need to iterate on this one.

Comment (HAKSOAT) [reply model_info/viwiki.articletopic.md:892]: What do you mean by 'iterate' in this case? Train more?

Comment (halfak) [reply model_info/viwiki.articletopic.md:892]: Woops.  Meant "iterate".  We might need to increase the vocab size or some other hyperparameter for Vietnamese.

**Pull request merged**